### DISCUSSION TITLE
Styles not generated on windows when using `outdir`

### DISCUSSION THREAD

**Discussion by romainhe** [chakra-ui/panda#2314] [General]
### Description

When I change the outdir value styles are not generated anymore, ex:
```ts
// The output directory for your css system
outdir: 'src/styled-system'
```

When I go back for the default value `outdir: 'styled-system'` (without re-generating the styled-system directiory in the new location) styles appear. 

### Link to Reproduction

https://github.com/romainhe/test-panda-css

### Steps to reproduce

1. Start a new next project
2. Setup panda
3. Set outdir to `src/styled-system`
4. Write styles on your page...

### JS Framework

Nextjs TS

### Panda CSS Version

0.33.0

### Browser

Chrome 122

### Operating System

- [ ] macOS
- [X] Windows
- [ ] Linux

**Comment by anubra266** [comment]
@romainhe did you rerun codegen after the change?

**Reply by romainhe** [reply]
yes I did, I'm preparing the reproduction repo

**Comment by segunadebayo** [comment]
There's a good chance that you need to set the `importMap` as well.

Kindly create a reproduction for us to help further.

**Reply by romainhe** [reply]
I've updated the description with the reproduction link.
I'm not sure what configuration to apply to importMap, a suggestion ?

**Reply by segunadebayo** [reply]
Consider setting import map to “ @/styled-system”

**Reply by Seldszar** [reply]
I have the same issue in the same configuration with Qwik and its default configuration (with `"src/styled-system"`) and for some reason, putting values like `"styled-system"`, `"~/styled-system"` and even `""` makes the extraction work again.

**Reply by IgnaceMaes** [reply]
I'm seeing similar behaviour in an Ember app where setting `outdir` to a subfolder (`app/styled-system`) stops styles from being generated. Using the root (`styled-systen`) works just fine.

**Reply by IgnaceMaes** [reply]
I should have tried the answer before commenting 😬 

Setting `importMap` in `panda.config.mjs` does the trick. Thank you!

```
{
  // ...
  outdir: "app/styled-system",
  importMap: "ember-pandacss-demo/styled-system",
}
```

**Reply by romainhe** [reply]
it was indeed the solution 👍 thx

**Comment by romainhe** [comment]
> Consider setting import map to “ @/styled-system”



> Consider setting import map to “ @/styled-system”
